apps/src/gamelab/animationLibrary.json

```diff
@@ -0,0 +1,332 @@
+{
```

Comment:
TODO: Add a 'header comment' to this JSON file explaining that it's autogenerated, and how to regenerate it.